Contributing.md

```diff
@@ -1,3 +1,84 @@
-Your contributions are always welcome!
+ # Contributing
 
-Please go through the active issues, before you plan to conribute.
\ No newline at end of file
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. 
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
```

Comment:
- Edit 2nd point as "Write a descriptive PR every time you make changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.

- Remove 3rd point because as of now, the project is not using any kind of versioning.

Reply:
@KishkinJ10 are you working on it?